search/src/Components/upload/upload_stepper.js

```diff
@@ -246,11 +248,20 @@ class UploadStepper extends React.Component {
         })
     }
 
+    handleCocoFormValidation(formKey, status){
+        this.setState(prevState => {
+            const newState = {coco_form_validation: {...prevState.coco_form_validation}}
+            newState.coco_form_validation[formKey] = status
+            console.log(newState)
```

Comment:
If we keep this in, it should have some words for context when looking at the log!!

Reply:
That was just for debugging. I'll remove it